scss/components/_button.scss

```diff
@@ -100,7 +100,7 @@ $button-transition: background-color 0.25s ease-out, color 0.25s ease-out !defau
   }
 
   -webkit-appearance: none;
-  border: 1px solid transparent;
+  border: $button-hollow-border-width solid transparent;
```

Comment:
Create a new variable `button-border-width` I guess ... This is not (only) for button hollow ??

Reply:
and also make `$button-hollow-border-width` equal to `$button-border-width`

Reply:
Do we need a button border width other than for styling hollow buttons?

Reply:
$button-hollow-border-width is not correct for non hollow buttons 

Create new variable `$button-border-width` and set to 1px
Edit `$button-hollow-border-width` to `$button-border-width` for avoiding code redundancy!

Reply:
okay!